src/ol/source/imagewmssource.js

```diff
@@ -312,7 +312,7 @@ ol.source.ImageWMS.prototype.getRequestUrl_ = function(extent, size, pixelRatio,
   }
   params['BBOX'] = bbox.join(',');
 
-  return goog.uri.utils.appendParamsFromMap(this.url_, params);
+  return this.url_ + '?' + ol.ext.querystring.stringify(params);
```

Comment:
No luck. :( Tried out using a full URI parser (https://github.com/petkaantonov/urlparser) but on second thought it seems too heavy.